site/learn/expert-advice/identity-basics/open-source-vs-commercial.md

```diff
@@ -0,0 +1,127 @@
+
+The peace of mind from having dedicated support can be one of the most important reasons to choose a commercial authentication provider over open source. At the end of the day, the buck stops with them. As noted above, you can often pay a company to support an open-source option, too.
+
+### Release Frequency
+
+Unlike with open-source providers, it may be difficult to know how often updates for commercial providers are released, especially if all of the provider's infrastructure lives in the cloud and updates don't require any action from the customer. They may release their software multiple times per day, or it may be months between public releases.
+
+This is one of the questions you can ask the sales team when evaluating different providers, but commercial providers don't always have the transparency associated with open-source providers. You can't always just go and look up previous release dates and research update progress yourself. However, some providers, such as FusionAuth, do offer [comprehensive release notes](https://fusionauth.io/docs/v1/tech/release-notes/) to aid you in your evaluation.
+
+### Relicensing
+
+Commercial providers tend to have either more obscure or more restrictive licenses. You won't be able to take the code and do anything you want with it. If you are building a typical web application, this shouldn't be an issue, but if you are embedding or redistributing your code with the auth system, that may violate the license. Ask your vendor. 
+
+Many of the issues of relicensing that come with open-source technologies don't apply to commercial authentication providers, because they usually provide a single license that goes with their product. They handle the licensing of any component parts in-house and usually have legal teams or at the very least an attorney that's reviewing all of the different aspects of software licensing.
+
+This is another thing you don't have to worry about with a commercial provider, rather than being responsible for reviewing all these licenses if you went with an open-source solution.
+
+### Maintenance Responsibility
+
+This is one of the main benefits of a commercial provider. They are responsible for all the maintenance of their product. Because you're paying for the product directly, this is one area you no longer have to worry about. You are trading money for time.
+
+However, unlike with open source, you have less freedom to make fixes yourself. So if you have a feature or a fix that you believe should be in the product, the most you can do is try to convince someone at the vendor to get that prioritized. With an open-source solution, you could fix it yourself, if you are willing to accept the maintenance burden.
```

Comment:
We should add the community aspect of FusionAuth with prioritizing features. And I agree that we should add an SOW section too